install-dev/theme/views/system.php

```diff
@@ -32,7 +32,7 @@
 <?php if ($this->tests['required']['success']): ?>
 	<h3 class="okBlock"><?php echo $this->translator->trans('PrestaShop compatibility with your system environment has been verified!', array(), 'Install'); ?></h3>
 <?php else: ?>
-	<h3 class="errorBlock"><?php echo $this->translator->trans('Oops! Please correct the item(s) below, and then click "Refresh" to test the compatibility of your new system.', array(), 'Install'); ?></h3>
+	<h3 class="errorBlock"><?php echo $this->translator->trans('Oops! Please correct the item(s) below, and then click "%refresh_label%" to test the compatibility of your new system.', array('%refresh_label%' => $this->trans('Refresh information', array(), 'Install')), 'Install'); ?></h3>
```

Comment:
Where did you find the 'Refresh information' string? I can't see it in Crowdin. Is it going to create a new one?

Reply:
My bad, it is in the following lines.

Reply:
Yes it didn't exist, so as we discussed I replaced the label text which was clearer this way